### ISSUE TITLE
Impossible to set up the map data path

### ISSUE THREAD

**Issue by isamuse2** [Akylas/alpimaps#10]
When I tap on the line "map data path" from the settings menu, nothing happens. Impossible to set up the map data path.
I've version 2.0.6 build 238 with Android 10

**Comment by farfromrefug** [comment]
@isamuse2 thanks i ll check locally and release a fix

**Comment by isamuse2** [comment]
@farfromrefug 
Thank you for the new release. I still have the bug!

**Comment by isamuse2** [comment]
My phone is OnePlus 5 with a stock rom (Android 10 + Oxygen 10.0.1).
When I click, nothing happens. The folder picker doesn't show up.



Le jeu. 4 mai 2023 à 18:36, farfromrefuge ***@***.***> a
écrit :

**Comment by farfromrefug** [comment]
@isamuse2 sorry for the very late answer. If you are still willing to test the latest version has some error reporting fixes. Hopefully now it should show an error when you try to Change the map data path

**Comment by isamuse2** [comment]
@farfromrefug No problem.
Now, I have the following error message
android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.OPEN_DOCUMENT_TREE flg=0x41 }

**Comment by farfromrefug** [comment]
@isamuse2 ok now i see. You dont have a "files" app ? Installing this will fix it https://github.com/zhanghai/MaterialFiles

**Comment by isamuse2** [comment]
@farfromrefug I've already material files installed!

**Comment by isamuse2** [comment]
I've installed v2.2.0 build 244
Now I've the message:
n.tW.once is not a function

**Comment by farfromrefug** [comment]
@isamuse2 i LL look at it

**Comment by farfromrefug** [comment]
@isamuse2 could you get a logcat of the error ?

**Comment by isamuse2** [comment]
Well... I'm not a developer. I'll try to find a way to do it.

**Comment by farfromrefug** [comment]
@isamuse2 there is an android app material log. But I think you need to be rooted. Otherwise you can do it from a computer with a little bit of setup.

**Comment by isamuse2** [comment]
Is it what you need?
[logcat_06-11-2023_16-38-01.txt](https://github.com/Akylas/alpimaps/files/11714880/logcat_06-11-2023_16-38-01.txt)

**Comment by farfromrefug** [comment]
@isamuse2 the format is weird. And i dont see the error sadly :s thank you for trying i will try to reproduce it here.

**Comment by farfromrefug** [comment]
@isamuse2 i reproduced and fixed the once issue. Thanks a lot for reporting all those errors! a new build 245 is coming soon

**Comment by isamuse2** [comment]
Thank you for your app. Compared to other maps app, AlpiMAps is easy to use. Particularly the huge number of maps and switching from one map to another.

I've noticed a typo in the French version. I did not find ressources for the translations, so I mention it here. In the menu "Astronomie" instead of "Astronoie" (missing m).

**Comment by farfromrefug** [comment]
@isamuse2 thanks for the kind words. Wondering are ypu using offline data? (Like explained in the readme ) asking because map data path is used for offline data. 
As for localization issues, thanks ! Files are in app/i18n

**Comment by isamuse2** [comment]
@farfromrefug yes I'm interested by offline data feature. But I have'nt yet checked your map generator. The great thing would be being able to use the same offline maps for several applications....

**Comment by farfromrefug** [comment]
@isamuse2 yes you can! My format is the common mbtiles one. Nothing special about it except map data is a little centered around hiking cycling(no trash can, trees ,...)
Also are you in France? I can share the pre generated data.
